### PR TITLE
update libsonata to release v0.1.10

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libsonata/package.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Libsonata(CMakePackage):
     git = "https://github.com/BlueBrain/libsonata.git"
 
     version('develop', branch='master', submodules=False, get_full_repo=True)
+    version('0.1.10', submodules=False, get_full_repo=True)
```
Comment (mgeplf) [inline var/spack/repos/builtin/packages/libsonata/package.py:20]: I don't understand the 'external' from below (like what was external?)

I suspect someone can update this?

Comment (matz-e) [reply var/spack/repos/builtin/packages/libsonata/package.py:20]: I don't recall what he issue there is/was, but those versions should go soon™

**Review by matz-e — APPROVED** [review]

**Comment by mgeplf** [comment]
> should go soon

In this PR, or a different one?

**Comment by matz-e** [comment]
> > should go soon
> 
> In this PR, or a different one?

doesn't matter to me, feel free to merge this one to accelerate deployment

**Comment by mgeplf** [comment]
Looking at the history; it was:
https://github.com/BlueBrain/spack/pull/1225#issuecomment-877218095

`Small update. It seems that the newest versions of libsonata depend on the ghc_filesystem submodule. As there is no Spack package for this dependency, the compilation fails with an error.`

It would be nice if that was part of the comment for next time.

**Comment by matz-e** [comment]
Retest this please

**Comment by matz-e** [comment]
Retest this please, oh almighty Jenkins!

**Review by matz-e — APPROVED** [review]

**Pull request merged**